Pipfile

```diff
@@ -28,13 +28,13 @@ flask-sqlalchemy = "*"
 alembic = "*"
 flask-script = "*"
 flask-migrate = "*"
-kerlescan = {editable = true,git = "https://github.com/beav/kerlescan.git",ref = "0.31"}
+kerlescan = {editable = true,git = "https://github.com/beav/kerlescan.git",ref = "0.33"}
```

Comment:
it looks like 0.32 is the latest right now. 0.33 fails when building

Reply:
yeah, this was anticipatory of the kerlescan update that is also taking place for this upgrade and after merging and tagging it, this could move forward.  Would we not need to tag a new kerlsecan when upgrading it to python 3.8?  If so, I'll back this down to 0.32.

Reply:
no kerlescan change is needed. IMO i'd leave this as-is in the PR